db/migrate/20210325164437_create_appointment_reminders.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAppointmentReminders < ActiveRecord::Migration[5.2]
+  def change
+    create_table :appointment_reminders, id: :uuid do |t|
+      t.date :remind_on, null: false
```

Comment:
Should this be a timestamp?

Reply:
Nope, because appointments are just a date as well...so the time a reminder is sent will be handled by our notification service.